features.bzl

```diff
@@ -24,7 +24,9 @@ _external_deps = struct(
     # https://github.com/bazelbuild/bazel/commit/970b9dda7cd215a29d73a53871500bc4e2dc6142
     module_extension_has_os_arch_dependent = ge("6.4.0"),
     # Whether repository_ctx#download has the block parameter, allowing parallel downloads (#19674)
-    download_has_block_param = ge("7.1.0")
+    download_has_block_param = ge("7.1.0"),
```

Comment:
I think this can be more precise, like objc_linking_info_migrated above there should be a "rolling" release where this is available in `7.1.0-pre.2023...`

(I was asked to do the same for my last contribution here in https://github.com/bazel-contrib/bazel_features/commit/9da382db4d8115d5f6575c4fea8b0c82afee4d65 )

Reply:
Strike that - rolling is 8.0.0-pre... at this point, and I see that we've only done that for features that landed in a major, not those that land in a minor.